### DISCUSSION TITLE
GitHub Copilot for Rstudio

### DISCUSSION THREAD

**Discussion by ran-codes** [Drexel-UHC/data-science#43] [Learning Resources]
**Copilot comes with a Pro GitHub Account   ( [ref](https://docs.github.com/en/billing/managing-billing-for-github-copilot/about-billing-for-github-copilot)).  All students/teachers can have free access to a Pro GitHub account.**

## Setup 

 - [ ] Upgrade GitHub account to 'Pro' by verifying your student/teacher status 
     - Verification link: https://education.github.com/discount_requests/teacher_application
     - Verication instructions:  https://makecode.com/github-teacher-verification
- [ ] Enable Copilot in Rstudio: https://www.youtube.com/watch?v=t7NrkAeosog&amp;ab_channel=ggnot2
   - [ ] Enable in global settings
   - [ ] connect to GitHub account:

**Comment by jlkephart** [comment]
Just did this, been coding for an hour, and I think my life is changed forever. Thanks Ran!
